lib/mesh.js

```diff
@@ -411,41 +467,66 @@ Mesh.prototype.initialize = Promise.promisify(function(config, callback){
           _this.getPackagedModules(config, callback);
         },
         function(callback) {
+
+          _this.startupProgress("got packaged modules", 70);
+
           if (!config.autoload || process.env.SKIP_AUTO_LOAD) {
             return callback();
           }
           _this.log.$$DEBUG('searched for autoload');
           _this.loadPackagedModules(config, null, callback);
         },
         function(callback) {
+
+          _this.startupProgress("got autoload modules", 75);
+
           if (!config.autoload || !process.env.SKIP_AUTO_LOAD) {
             _this.log.$$DEBUG('initialized autoload');
           }
           _this._loadComponentSuites(config, callback);
         },
         function(callback) {
+
+          _this.startupProgress("loaded component suites", 75);
```

Comment:
ok, do you really want this to change, I repeat the value because this is part of the previous 75 - so just ensuring that if someone opened the loader page, at this point they see 75%, do you want me to make it 76% ?